CoinControlTests/CoinControlTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CoinControlTests.swift
+//  CoinControlTests
+//
+//  Created by Md. Asadul Islam on 29/2/24.
+//
+
+import XCTest
+
+final class CoinControlTests: XCTestCase {
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>warning: (blankLinesAtStartOfScope) Remove leading blank line at the start of a scope.